src/main/java/eu/ha3/presencefootsteps/sound/acoustics/VaryingAcoustic.java

```diff
@@ -45,16 +61,15 @@ public void playSound(SoundPlayer player, LivingEntity location, State event, Op
             return;
         }
 
-        float volume = this.volume.random(player.getRNG());
-        float pitch = this.pitch.random(player.getRNG());
+        final float volume = inputOptions.containsKey("gliding_volume")
+                ? this.volume.on(inputOptions.get("gliding_volume"))
+                : this.volume.random(player.getRNG());
 
-        if (inputOptions.containsKey("gliding_volume")) {
-            volume = this.volume.on(inputOptions.get("gliding_volume"));
-        }
-        if (inputOptions.containsKey("gliding_pitch")) {
-            pitch = this.pitch.on(inputOptions.get("gliding_pitch"));
-        }
+        final float pitch = inputOptions.containsKey("gliding_pitch")
+                ? this.pitch.on(inputOptions.get("gliding_pitch"))
+                : this.pitch.random(player.getRNG());
 
-        player.playSound(location, soundName, volume, pitch, getOptions());
+        player.playSound(location, soundName, volume, pitch, Options.EMPTY);
```

Comment:
Why is this changed to Options.EMPTY here rather than calling getOptions()?

Reply:
I removed the #getOptions method while converting VaryingAcoustic & DelayedAcoustic to Records since VaryingAcoustic forced Options#EMPTY anyway. I've now rewritten the #playSound method to share code between the two classes like before.